Improve layout of ctf_map GUI

ClobberXD opened the pull request:
- Improve layout consistency.
- Add a text field for `initial_stuff`.

![screenshot_20181111_163625](https://user-images.githubusercontent.com/36130650/48312148-67aab880-e5d0-11e8-9c28-89013deb95af.png)

AKryukov92 commented:
Do we have some kind of tutorial about this formspec?
Which delimiter should be used if I want to define multiple starting items?

ClobberXD commented:
> Do we have some kind of tutorial about this formspec?

https://youtu.be/orBsC9wViUw

> Which delimiter should be used if I want to define multiple starting items?

Multiple items are comma-separated. I initially did want to add this clarification, but there's just not enough space. Should I increase the formspec height and re-add it?

rubenwardy commented:
Maybe also add horizontal and vertical rules to split up sections? Can be done with a stretched `background[]`

rubenwardy commented on mods/ctf_map/map_maker.lua:
```diff
@@ -149,30 +151,33 @@ local function show_gui(name)
-		"field[4.15,7.8;3.75,1;author;Author;", minetest.formspec_escape(mapauthor), "]",
+		"label[0,5.2;4. Meta Data]",
+		"field[0.4,6.2;8.5,1;title;Title;" , minetest.formspec_escape(maptitle), "]",
+		"field[0.4,7.3;8.5,1;initial_stuff;Initial Stuff (optional): ",
```
```
Stuff to give on (re)spawn, comma separated
```

ClobberXD replied:
Added. Please review. :)

ClobberXD commented:
I'd love to add some separators, thanks for the suggestion. :)

ClobberXD commented:
- Improved the label of `initial_stuff` field.
- Added separators using `box[]` element, with the colorstring `#111111BB`.

Updated screenshot in first post.
